PluginCore/PluginCore/FRService/FRSearch.cs

```diff
@@ -608,6 +609,45 @@ private List<SearchMatch> SearchSource(string src, int startIndex, int startLine
             return results;
         }
 
+        private bool LookupRegex(string ba, ref int i)
+        {
+            if (PluginBase.CurrentProject == null || 
```

Comment:
We certainly need to lookup regexes, I just mean that we should not test the project language.

If you want to do Haxe-specific lookup it would make more sense to check the file extension (if that's an information we have).